### ISSUE TITLE
Multiple implementations of same method should give error

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#514]
```abap
  METHOD moo.
  ENDMETHOD.
  METHOD moo.
  ENDMETHOD.
```

**Comment by larshp** [comment]
https://rules.abaplint.org/method_implemented_twice/

closing
